Tensile/SolutionLibrary.py

```diff
@@ -31,6 +31,12 @@
 from .SolutionStructs import Solution as OriginalSolution
 from .Utils import state
 
+architectureList = ['fallback', 'gfx803', 'gfx900', 'gfx906', 'gfx908', 'gfx90a',
+                    'gfx940', 'gfx941', 'gfx942', 'gfx1010', 'gfx1011', 'gfx1012',
+                    'gfx1030', 'gfx1031', 'gfx1032', 'gfx1034', 'gfx1035', 'gfx1100', 
+                    'gfx1101', 'gfx1102']
+solutionIndexMap = {architectureName:int(offset*pow(2,16)) 
```

Comment:
By the way, is 16bit large enough to cover each architecture?
What is the current largest number?

Reply:
The largest solution set is currently gfx90a with 43056.

Reply:
Hmm.  It is almost reaching the max.
Maybe better to increase bits for index a bit.
  18 or 20 bit for index for each arch
  14 or 12 bit for arch
or something like this.

Reply:
We can do 14 bit arch + 18 bit solution indices.